JeongJaeHeon/app.js

```diff
@@ -0,0 +1,58 @@
+const http = require('http');
+
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const { DataSource } = require('typeorm')
+
+const app = express();
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE,
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized")
+    })
+    .catch(() => {
+        console.log("Data Source has not been initialized")
+    })
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('combined'));
+
+app.get('/ping',(req, res, next) => {
+    res.status(200).json({message : 'pong'});
+});
+
+app.post('/users/create', async (req, res, next) => {
```

Comment:
post 메소드는 주로 새로운 리소스를 생성할 때 사용되는 메소드입니다.
이미 메소드 이름에 충분히 의미가 부여되었으니 URI는 요청에 맞는 자원을 기준으로 작성해 주세요.

```suggestion
app.post('/users', async (req, res, next) => {
```